test/net_http_test.rb

```diff
@@ -446,7 +446,7 @@ def open_circuit!(hostname: nil, toxic_port: nil, toxic_name: "semian_test_net_h
       uri = URI("http://#{hostname}:#{toxic_port}/200")
       http.raw_semian_options[:error_threshold].times do
         # Cause error error_threshold times so circuit opens
-        Toxiproxy[toxic_name].downstream(:latency, latency: 150).apply do
+        Toxiproxy[toxic_name].downstream(:latency, latency: 500).apply do
```

Comment:
This was flaky. The timeout is 100ms, but I expect it's not exactly 100ms 🙃 This failed quite a lot locally.